containers/Comments/Comment.js

```diff
@@ -89,6 +89,18 @@ export const Comment = (props) => {
     setHasExcerpt(e.nativeEvent.lines.length > NUM_OF_LINES);
   }, []);
 
+  function handleOptions(subcomment) {
+    if (subcomment !== undefined){
```

Comment:
can subcomment be null or 0? if not maybe just `if (!subcomment)` ?

Reply:
it's undefined if called from comment and defined if called from a subcomment, so I think we need the undefined check?

Reply:
yes in that case undefined